benchmark/inference/inference_benchmark.py

```diff
@@ -0,0 +1,126 @@
+import argparse
+import copy
+from timeit import default_timer
+
+import torch
+from utils import get_dataset, get_model
+
+from torch_geometric.loader import NeighborLoader
+from torch_geometric.nn import PNAConv
+
+supported_sets = {
+    'ogbn-mag': ['rgat', 'rgcn'],
+    'reddit': ['edge_conv', 'gat', 'gcn', 'pna_conv'],
+    'ogbn-products': ['edge_conv', 'gat', 'gcn', 'pna_conv'],
+}
+
+
+def run(args: argparse.ArgumentParser) -> None:
+
+    device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+    progress_bar = True
+
+    print('BENCHMARK STARTS')
+    for dataset_name in args.datasets:
+        print(f'Dataset: {dataset_name}')
+        dataset = get_dataset(dataset_name, args.root)
+
+        hetero = True if dataset_name == 'ogbn-mag' else False
+        mask = ('paper', None) if hetero else None
+        degree = None
+
+        data = dataset[0].to(device)
+        inputs_channels = data.x_dict['paper'].size(
+            -1) if hetero else dataset.num_features
```

Comment:
Lines 29 and 33-34 seem specific to `ogbn-mag` (instead of `hetero`), and will likely break if we add more heterogeneous datasets in the future. Can we condition on `if dataset_name == 'ogbn-mag'` instead?